zjit/src/hir_type/gen_hir_type.rb

```diff
@@ -75,6 +75,7 @@ def base_type name
 base_type "Set"
 base_type "Regexp"
 base_type "Class"
+base_type "Module"
```

Comment:
I wonder if we should explicitly mark Class as subclassing Module in the type hierarchy

Reply:
Updated 👍